### PR TITLE
lair-keystore update 0.1.3

### PR DESCRIPTION
**Pull request by fayeed**
With the new version `lair-keystore` now expects passphrase with it in the previous version we would save the passphrase in `conductor-config.yaml` and it was `foobar` hardcoded. This also changes the flow of how the executor starts different services. With the new version, `holohain` & `lair-keystore` only start when the agent unlocks.

Previous flow (Had hardcoded passphrase `foobar`):
`Start Perspectivsm Core -> Start IPFS -> Start Lair Keystore -> Start Holohchain -> Start Graphql Server`

New Flow (Uses the agent passphrase):
`Start Perspectivsm Core -> Start IPFS  -> Start Graphql Server  -> User unlocks the agent -> Start Lair Keystore -> Start Holohchain`

### CODE REVIEW

**Comment by jdeepee** [comment]
Lets also keep an eye out an ensure that lair-keystore process still gets killed correctly on executor close with the SIGINT command we send

File: src/core/storage-services/Holochain/HcExecution.ts [hunk 1]
```diff
@@ -30,6 +32,7 @@ export interface ConductorConfiguration {
     useProxy: boolean,
     useLocalProxy: boolean;
     useMdns: boolean;
+    connectionUrl: string;
```
Comment (lucksus) [inline src/core/storage-services/Holochain/HcExecution.ts:35]:
```suggestion
    lairConnectionUrl: string;
```
?

File: src/tests/integration.test.ts [hunk 2]
```diff
@@ -97,6 +97,14 @@ export class TestContext {
...
     }
 
+    get aliceCore() : PerspectivismCore {
+      return this.#aliceCore!
+    }
+
+    get bobCore(): PerspectivismCore {
+      return this.#bobCore! 
+    }
+
```
Comment (lucksus) [inline src/tests/integration.test.ts:108]: I don't see these being used anywhere...

**Review by lucksus — COMMENTED** [review]
I have two minor remarks. Looks good otherwise!

**Comment by fayeed** [comment]
@lucksus Fixed the comments.

**Review by lucksus — APPROVED** [review]

**Pull request merged**